Connector feign client not working

The new camunda rest client in the connector does fail when parsing a exception from the engine. See below log.

```
java.util.concurrent.CompletionException: java.lang.NoClassDefFoundError: com/fasterxml/jackson/module/kotlin/ExtensionsKt
    at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
    at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
    at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
    at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.NoClassDefFoundError: com/fasterxml/jackson/module/kotlin/ExtensionsKt 
```

--- a/digiwf-connector/digiwf-camunda-connector-service/pom.xml
+++ b/digiwf-connector/digiwf-camunda-connector-service/pom.xml
@@ -42,6 +42,19 @@
             <groupId>de.muenchen.oss.digiwf</groupId>
             <artifactId>digiwf-spring-logging-and-tracing</artifactId>
         </dependency>
+        <!-- json etc. -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-kotlin</artifactId>
+        </dependency>
 
         <!-- Security -->
         <dependency>

